tldw_Server_API/app/core/AuthNZ/auth_principal_resolver.py

```
@@ -0,0 +1,308 @@
+                api_key_id=None,
+            )
+            ctx = _build_context(principal, request)
+            try:
+                request.state.auth = ctx
+            except Exception:
+                pass
+            return principal
+    except HTTPException:
+        # Propagate auth failures as-is
+        raise
+    except Exception as exc:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)